<launch>


<arg name="robot_id" default="aristos"/>
<arg name="prefix" default="aristos_" />
<arg name="pub_odom_tf" default="false" />
<arg name="config_dir" default="$(find roboteq_motor_controller_driver)/config/query_noimu.yaml"/>
<arg name="use_imu" default="false"/>
<arg name="imu_topic" default="/aristos/imu/data"/>

<group ns="aristos/wheel/noimu">


    <node name = "diff_odom_noimu" pkg = "roboteq_motor_controller_driver" type = "diff_odom" output = "screen">
        <remap from="abs_hall_count" to="/aristos/wheel/abs_hall_count"/>
        <remap from="hall_count" to="/aristos/wheel/hall_count"/>
    </node>

</group>




 
</launch>
